badges/index.php

```diff
@@ -59,21 +59,30 @@
             <title name="title">Osekai Badges • All of osu!\'s Badges!</title>
             <meta name="keywords" content="osekai,osu,osu!,osu!game,osugame,game,video game,award,badge,badges">
             <meta property="og:url" content="/badges" />';
-        }
-        frontend();
+    }
+    frontend();
     echo $meta;
 
     ?>
     <?php
 
     font();
     css();
-            notification_system();
+    notification_system();
     fontawesome();
     ?>
 </head>
 
 <body>
+
+    <script>
+        const BADGES =

```

Comment:
please don't do this. we rewrote the entire website specifically to stop printing data directly onto the site.
what is most important, is time from click to first draw, this probably increases that quite a bit. i don't care how long badges take to pop up, as long as the navbar and such is there sooner

Reply:
donedid